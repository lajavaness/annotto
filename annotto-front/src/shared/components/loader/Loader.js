import { Spin } from 'antd'
import React from 'react'

import * as Styled from './__styles__/Loader.styles'

const Loader = () => (
	<Styled.Root data-testid={'__loader__'}>
		<Spin size="large" />
	</Styled.Root>
)

export default Loader
