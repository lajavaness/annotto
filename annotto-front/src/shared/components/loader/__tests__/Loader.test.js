import { render } from '@testing-library/react'
import React from 'react'

import Loader from 'shared/components/loader/Loader'

const defaultProps = {}
const getInstance = (props = {}) => <Loader {...defaultProps} {...props} />

describe('AnimatedLoader', () => {
	it('matches snapshot', () => {
		const { asFragment } = render(getInstance())
		expect(asFragment()).toMatchSnapshot()
	})
})
