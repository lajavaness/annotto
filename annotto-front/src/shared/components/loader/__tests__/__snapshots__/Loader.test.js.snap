// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedLoader matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBQB dbZmsR"
    data-testid="__loader__"
  >
    <div
      class="ant-spin ant-spin-lg ant-spin-spinning"
    >
      <span
        class="ant-spin-dot ant-spin-dot-spin"
      >
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
